server/libs/userlibs.js

```diff
@@ -148,4 +149,100 @@ const logOut = function(request) {
     })
 }
 
-module.exports = {register, logIn, updateUser, logOut}
\ No newline at end of file
+const adminDeActivateUser = function(body) {
+    let id = body.uuid;
+    return new Promise((resolve,reject) => {
+        User.findOne({uuid : id}).then((user) => {
```

Comment:
Don't need to call find one. use get user function already created. Try to use async function to code properly and neat

Reply:
Done